File: Python/conditional.py
#!/usr/bin/env python

# Demonstrate if statement

from random import randint

answer = randint(0,9)
question = 'What number am I thingking of? '

print ("Guess a number between 0 and 9")

while True:
    guess = int(input(question))

    if guess < answer:
        print ("Too Low")
    elif guess > answer:
        print ("Too High")
    else:
        print ("Great Guess!!!")
        break

File: Java/Conditional.java
// Demonstrate if statement

import java.util.Scanner;
import java.util.concurrent.ThreadLocalRandom;

public class Conditional {
    public static void main(String [] args) {

        // Class to read from standard input
        Scanner in = new Scanner(System.in);
    
        // Generate a random number
        int answer = ThreadLocalRandom.current().nextInt(0, 10);

        String question = "What number am I thinking of? ";
    
        System.out.println("Guess a number between 0 and 9");
    
        while(true){
            // Ask for input
            System.out.printf(question);
            int guess = in.nextInt();
    
            // Check to see if the input matches the random number
            if( guess < answer){
                System.out.println("Too Low");
            } else if( guess > answer ){
                System.out.println("Too High");
            } else {
                System.out.println("Great Guess!!!");
                break;
            }
        }
    }
}

